src/sql/workbench/services/errorMessage/browser/errorMessageDialog.ts

```diff
@@ -88,7 +89,7 @@ export class ErrorMessageDialog extends Modal {
 	}
 
 	private createStandardButton(label: string, onSelect: () => void): Button {
-		let button = this.addFooterButton(label, onSelect, 'right', true);
+		let button = this.addFooterButton(label, onSelect, 'right', false);
```

Comment:
are we sure there aren't side-effects with this change in other error dialogs?  I think this is to make it not a "secondary" button?

Reply:
Yes, since this applies to the action buttons added externally other than the default OK button, they should be Primary IMO since we update the default OK button label to 'Close'.

Reply:
I would suggest we not change default behavior for existing dialogs unless you've been able to go through and review all uses of the dialogs - if people are used to having "close" be the primary button then changing it may cause them to run the wrong action by accident.

Reply:
This change only affects callers who are passing custom 'actions', and looking at all references of `ErrorMessageService.showDialog`, there is no other call that passes 'actions' that would change this behavior. This behavior will only be observed with 'Run Kinit' action (above the new SSL cert validation failure handling) which doesn't seem like a big impact.

Reply:
Ah, so it is! Then yeah this seems fine to me.